lib/components/DepositorUI.jsx

```diff
@@ -1,30 +1,39 @@
 import React from 'react'
 
+import { AwardPrizeCard } from 'lib/components/AwardPrizeCard'
 import { Card } from 'lib/components/Card'
 import { Collapse } from 'lib/components/Collapse'
 import { DepositUI } from 'lib/components/DepositUI'
 import { PrizeCard } from 'lib/components/PrizeCard'
 import { WithdrawUI } from 'lib/components/WithdrawUI'
-import { DepositorOddsCards } from 'lib/components/DepositorOddsCards'
 import { RelativeInternalLink } from 'lib/components/RelativeInternalLink'
-import { PoolDepositorStats } from 'lib/components/PoolStats'
+import { CompleteStatsList, UsersStats } from 'lib/components/PoolStats'
+import { useTimeLeftBeforePrize } from 'lib/hooks/useTimeLeftBeforePrize'
 
 export const DepositorUI = (props) => {
+  const { timeRemaining } = useTimeLeftBeforePrize()
+
   return (
     <>
       <PrizeCard className='mb-4' />
-      {/* <DepositorOddsCards /> */}
+      {!timeRemaining && <AwardPrizeCard />}
       <Card>
         <Collapse title='Deposit to win' openOnMount>
           <DepositUI />
         </Collapse>
       </Card>
-      <PoolDepositorStats />
+      <Card>
+        <CompleteStatsList />
```

Comment:
These lists should be wrapped with `CardDetailsList`, can we make more components in `PoolStats.jsx` like `PoolStats` (which should probably be named `PoolStatsCard` and `PoolStatsCard` renamed to `PoolStatsCardLayout`)